usaspending_api/tests/test_award_index_elasticsearch.py

```diff
@@ -0,0 +1,590 @@
+import pytest
+
+from model_mommy import mommy
+
+
+@pytest.fixture
+def award_data_fixture(db):
+    mommy.make("references.LegalEntity", legal_entity_id=1)
```

Comment:
We're trying to remove all references to `legal_entity` before it is removed. It is necessary here?

Reply:
nope